app/presenters/confirmation_email_addresses_presenter.rb

```diff
@@ -0,0 +1,26 @@
+class ConfirmationEmailAddressesPresenter < Struct.new(:claim)
+  delegate :primary_claimant, :representative, :secondary_claimants, to: :claim
+
+  def self.for(claim)
```

Comment:
Makes sense. Originally I had this as a service object(without a suffix) & felt a little more expressive regarding what it returns.